puma/assets/configuration/spec.yaml

```
@@ -0,0 +1,15 @@
+name: puma
+files:
+- name: puma.yaml
+  options:
+  - template: init_config
+    options: []
```

```suggestion
    options:
        - template: init_config/http
        - template: init_config/default

```

let's include the templates for http options and default options